package.json

```diff
@@ -10,9 +10,11 @@
     "lint": "eslint lib test util"
   },
   "dependencies": {
+    "bignumber.js": "^6.0.0",
     "request": "^2.81.0",
+    "retry": "^0.10.1",
     "underscore": "^1.8.3",
-    "ws": "^3.1.0"
+    "ws": "^4.0.0"
```

Comment:
Since this PR also has some age behind it, would appreciate a quick check that the ws bump to 4.0.0 is still working fine.